terraform/eks/container-insights-agent/logs_sample_fargate.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flblogproducer2eks
```

Comment:
Let's rename this to `app-container` in 5 places. This is not related to Fluent Bit. It was added only for testing purposes.